components/director/pkg/graphql/webhook_validation.go

```diff
@@ -78,3 +82,13 @@ func (i WebhookInput) Validate() error {
 		validation.Field(&i.Auth),
 	)
 }
+
+func isOutTemplateMandatory(webhookType WebhookType) bool {
```

Comment:
[minor suggestion] what if this check uses `switch`
```suggestion
switch webhookType {
	case WebhookTypeRegisterApplication, WebhookTypeUnregisterApplication:
		return true
	default:
		return false
	}
```